### PR TITLE
Player score

### LINKED ISSUE #60: Display score

### LINKED ISSUE #73: Multiplayer Functionality
- [x] Display wins issue complete

### PR DESCRIPTION
**Pull request by GreyMatteOr**
## Description
Clearly and concisely describe the feature.

Integration. Tons of small UI features:

- Loading screen enhancement for both gnerator as well as guesser
- A game-end overlay before resetting
- Jumping to bottom of chat
- some icons to indicate the status of a player
- player cards displaying score and remaining `tries`
- disabling the guess button when a player has won

## Where To Start

Live on local

## Affected areas of application

The gamepage, primarily

## How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
- [x] In window Browser
- [x] Test Suite

## Relevant Tickets

closes #73 
closes #60

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -24,11 +24,13 @@ export class App extends Component{
...
       isOver: false,
       hasGenerator: null,
+      hasWord: false,
       userName: this.props.userName || 'guest',
       rooms: this.props.rooms || [],
       numOnline: 'calculating the number of',
       attemptsLeft: 0,
-      playerNames: []
+      playerNames: [],
+      winners:['--REDACTED--']
```
Comment (holladayian) [inline src/App.js:33]: What is the reason for `'--REDACTED--'`? Not complainin', just askin'.

File: src/Chat/Chat.js [hunk 2]
```diff
@@ -16,7 +16,8 @@ export class Chat extends Component {
     this.messagesEnd.current.scrollIntoView({ behavior: "smooth" });
   }
 
-  componentDidUpdate() {
+  componentDidUpdate = () => {
+    console.log('trigger')
```
Comment (holladayian) [inline src/Chat/Chat.js:20]: I think I removed this in mine. Hopefully it isn't a merge conflict.

File: src/Chat/Chat.scss [hunk 3]
```diff
@@ -20,7 +20,7 @@
   min-height: 350px;
   width: 100%;
   border-radius: 5%;
-} 
+}
```
Comment (holladayian) [inline src/Chat/Chat.scss:23]: oh yeah

File: src/WordSelector/WordSelector.scss [hunk 4]
```diff
@@ -35,4 +35,4 @@
   to {
     transform: rotate(360deg);
   }
-}
\ No newline at end of file
+}
```
Comment (holladayian) [inline src/WordSelector/WordSelector.scss]: oh yeah

**Review by holladayian — APPROVED** [review]

File: src/App.js (hunk 1 above)
Comment (GreyMatteOr) [reply src/App.js:33]: A somewhat humorous way to error - test 😆

**Pull request merged**